pyproject.toml

```diff
@@ -44,6 +44,10 @@ dev = [
     "pytest-testmon",
     "pytest-xdist",
     "pytest-icdiff",
+    "sphinx",
+    "sphinxcontrib-django",
+    "sphinx-last-updated-by-git",
+    "sphinx-rtd-theme",
```

Comment:
Could you execute `./tools/update_dependencies.sh --only=pip` to add these dependencies to the pinned versions as well?